app/logic/searchUsers.py

```diff
@@ -9,18 +10,36 @@ def searchUsers(query):
 
     firstName = splitSearch[0] + "%"
     lastName = " ".join(splitSearch[1:]) +"%"
+    results = None
+    searchId = None
 
     if len(splitSearch) == 1: #search for first or last name
-        results = User.select().where(User.isStudent & (User.firstName ** firstName | User.lastName ** firstName))
+        if searchGroup == "insideParticipant":
```

Comment:
Doesn't this break everything? it's pretty weird that in order to search for students (in any context) you would have to pass 'insideParticipant' instead of 'student'

Reply:
We typically had two search groups, insideParticipant and outsideParticipant. However, I have changed insideParticipant to student. If we agree that insideParticipant are people with B# then it becomes easy in this scenario.